Crash during sync

Looks like https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/sync/pending_blocks_queue.go#L53 is coming back `nil`:

```
{"headRoot":"0x9d06c99ae9ed0e9bff9a1acf45ae31996de870269586d7c3d1907dd4a8ec312b","level":"debug","msg":"Saved new head info","prefix":"blockchain","slot":4896,"time":"2020-01-10T16:37:32Z"}
{"attestations":128,"deposits":0,"level":"info","msg":"Finished applying state transition","prefix":"blockchain","slot":4896,"time":"2020-01-10T16:37:32Z"}
{"level":"info","msg":"Processed ancestor block with slot 4896 and cleared pending block cache","prefix":"sync","time":"2020-01-10T16:37:32Z"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x11b4970]

goroutine 116 [running]:
github.com/prysmaticlabs/prysm/beacon-chain/sync.(*Service).processPendingBlocks(0xc0014bdc80, 0x1b52280, 0xc06e3aec00, 0x0, 0x0)
	beacon-chain/sync/pending_blocks_queue.go:54 +0x3e0
github.com/prysmaticlabs/prysm/beacon-chain/sync.(*Service).processPendingBlocksQueue.func1()
	beacon-chain/sync/pending_blocks_queue.go:28 +0x3c
github.com/prysmaticlabs/prysm/shared/runutil.RunEvery.func1(0xc001632000, 0x2547058, 0x5b, 0xc00162a040, 0x1b521c0, 0xc0014ea240)
	shared/runutil/every.go:22 +0xec
created by github.com/prysmaticlabs/prysm/shared/runutil.RunEvery
	shared/runutil/every.go:17 +0x136
```

--- a/beacon-chain/sync/pending_blocks_queue.go
+++ b/beacon-chain/sync/pending_blocks_queue.go
@@ -51,6 +51,10 @@ func (r *Service) processPendingBlocks(ctx context.Context) error {
 
 		r.pendingQueueLock.RLock()
 		b := r.slotToPendingBlocks[uint64(s)]
+		// skip if block does not exist
+		if b == nil || b.Block == nil {
+			continue
+		}
 		inPendingQueue := r.seenPendingBlocks[bytesutil.ToBytes32(b.Block.ParentRoot)]
 		r.pendingQueueLock.RUnlock()
 

--- a/beacon-chain/sync/pending_blocks_queue.go
+++ b/beacon-chain/sync/pending_blocks_queue.go
@@ -51,7 +51,7 @@ func (r *Service) processPendingBlocks(ctx context.Context) error {
 
 		r.pendingQueueLock.RLock()
 		b := r.slotToPendingBlocks[uint64(s)]
-		// skip if block does not exist
+		// Skip if block does not exist.
 		if b == nil || b.Block == nil {
 			continue
 		}

